mercadopago/px-ios
# (Changed)[PXN-3089] - Changed discounts icon in Summary to match charges icon

## What has changed?

Changed discounts icon in the summary to match charges icon (as expected by UX).

Now we will not get the discount icon from the checkout response, but by our Resources Manager. 

| Before | After |
|:-:|:-:|
| ![image](https://user-images.githubusercontent.com/61248081/151049164-895e950f-e43e-487a-a4b6-109547d3eaeb.png) | ![image](https://user-images.githubusercontent.com/61248081/151048916-162b39f1-38c2-4deb-84f4-09690d01f454.png) |

## How to test:

Build a checkout instance with discount and charges, and check for same helper icons.
